go.mod

```diff
@@ -1,11 +1,12 @@
 module github.com/shimmerglass/http-mirror-pipeline
 
-go 1.14
+go 1.15
```

Comment:
Why update the Go version at the same time?

Reply:
I was already modifying the go.mod to add prometheus so just bumped the version to stay up to date.